### PR TITLE
Too many things sorry, i don't have idea for the title

### PR DESCRIPTION
**Pull request by Altansar69**
Redpoint and I have done a lot of updating. We haven't done anything for several days, so I'm taking this opportunity to make a pull request. If you accept the merge, can you do a release? Thx.

### CODE REVIEW

File: config.html [hunk 1]
```diff
@@ -228,7 +231,7 @@
                 <tr colspan="2">
                     <td>
                         <input type="checkbox" id="script_enabled" onclick="enableScript();"
-                            data-originalValue=""></input>
```
Comment (SubZeroPL) [inline config.html:231]: Why did you remove closing of input tag?

Comment (Reddiepoint) [reply config.html:231]: Hi SubZeroPL,

I removed the closing tags because they aren't actually necessary. See: [https://www.w3schools.com/tags/tag_input.asp](https://www.w3schools.com/tags/tag_input.asp)

Comment (SubZeroPL) [reply config.html:231]: Oh, I didn't know that, thanks for clarification. Then it looks fine.

**Review by SubZeroPL — APPROVED** [review]
